scripts/release/atomic_pipeline.py

```diff
@@ -49,14 +49,12 @@ def build_image(
     span.set_attribute("mck.platforms", build_configuration.platforms)
 
     # Build docker registry URI and call build_image
-    image_full_uri = f"{build_configuration.registry}:{build_configuration.version}"
-
-    logger.info(
-        f"Building {image_full_uri} for platforms={build_configuration.platforms}, dockerfile args: {build_args}"
-    )
+    tags = [f"{build_configuration.registry}:{build_configuration.version}"]
+    if build_configuration.latest_tag:
```

Comment:
Is it possible to handle tags as a list in the build configuration instead of treating `latest` as a special case ?